mix.exs

```diff
@@ -60,6 +60,7 @@ defmodule Skate.MixProject do
       {:ex_aws, "~> 2.1"},
       {:ex_doc, "~> 0.27", only: :dev, runtime: false},
       {:ex_machina, "~> 2.7.0", only: :test},
+      {:excellent_migrations, "~> 0.1", only: [:dev, :test], runtime: false},
```

Comment:
question(non-blocking): why do we need this in `:test`?

Reply:
Huh... I just grabbed that line from the `excellent_migrations` docs. Let's see if CI likes it if we move it to only be `:dev`.

Reply:
CI liked it!